flatpak/flatpak-node-generator.py

```diff
@@ -0,0 +1,1839 @@
+#!/usr/bin/env python3
```

Comment:
`flatpak-node-generator` is updated often to catch up with nodejs ~quirks~ updates. If you want it to be part of you app repo, you probably better of adding flatpak-builder-tools as a submodule instead of committing it directly in the app tree.